plugin/evm/vmsync/registry_test.go

```diff
@@ -162,7 +229,10 @@ func TestSyncerRegistry_RunSyncerTasks(t *testing.T) {
 				require.NoError(t, registry.Register(syncerConfig.name, mockSyncer))
 			}
 
-			err := registry.RunSyncerTasks(context.Background(), &client{})
+			ctx, cancel := utilstest.NewTestContext(t)
```

Comment:
`utilstest.NewTestContext` is really opaque, adding nothing more than a 30-second deadline (or the test's own one) while forcing the reader to check that the new `Context` doesn't do anything that will interfere with the test itself. I would just use `context.Background()` or specify the deadline yourself (also just one line, but completely explicit).